hydra-jobs.nix

```diff
@@ -54,6 +54,5 @@ let
   ciJobsets = stripAttrsForHydra (filterDerivations ci);
 in
 traceNames "" (ciJobsets // {
-  required = derivationAggregate "required-marlowe" ciJobsets;
```

Comment:
A little magic needed in order to merge the new hydraJobs with the old hydraJobs.
The `required` job is defined in flake.nix now